solver/template/tpl_butcher_methods.hxx

```diff
@@ -78,6 +78,18 @@ l{{ rk.id }}( Exp_t exp_ , double tol=ponio::default_config::tol )
 
 {% endfor %}
 
+/**
+ * @brief Type of tuple that contains all classical explicit Runge-Kutta methods of ponio
+ */
+template <typename value_t>
+using erk_tuple = std::tuple< {{ list_erk | sformat("{}_t<value_t>", attribute="id") | join(", ") }} >;
+
+/**
+ * @brief Type of tuple that contains all Lawson methods of ponio
+*/
+template <typename value_t, typename Exp_t>
+using lrk_tuple = std::tuple< {{ list_erk | sformat("decltype(l{}_t<value_t, Exp_t>)", attribute="id") | join(", ") }} >;
```

Comment:
Maybe you can use more readable names such as `explicit_rk_list`, `lawson_rk_list`, ...

Reply:
`explicit_rk_list` or `explicit_rk_tuple` ? (same for Lawson or expRK methods)